indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -34,68 +34,108 @@
                         {% for abstract in abstracts %}
                             <tr class="i-table abstract-row"
                                 data-friendly-id="{{ abstract.friendly_id }}"
-                                data-title="{{ abstract.as_legacy.getTitle() }}">
+                                data-title="{{ abstract.title }}">
                                 <td class="i-table id-column">
-                                    <span class="vertical-aligner">
-                                        <input type="checkbox" class="select-row" name="abstract_id"
-                                               value="{{ abstract.id }}">
-                                    </span>
+                                    <input type="checkbox" class="select-row" name="abstract_id"
+                                           value="{{ abstract.id }}">
```

Comment:
Well, what's the convention in HTML? We don't have a rule and in any case, it's above 180 chars.